File: Project/Remix_pwa/app/Models/implementations/DivElement.ts
import { HtmlElement } from "../interfaces/HtmlElement";

export class DivElement extends HtmlElement {
    override get Name(): string { return "div"; }

    constructor() {
        super();
        this._style.set("width", "100px");
        this._style.set("height", "100px");
    }
}

File: Project/Blazor_pwa/Models/Implementations/DivElement.cs
using Blazor_pwa.Models.Interfaces;

namespace Blazor_pwa.Models.Implementations
{
    public class DivElement : HtmlElement
    {
        public override string Name => "div";

        public DivElement() : base()
        {
            Attributes.Add("", "");
        }
    }
}